pkg/api/getambassador.io/crds.yaml

```diff
@@ -1871,6 +1871,8 @@ spec:
               flush_interval_byte_size:
                 type: integer
               flush_interval_time:
+                description: 'TODO(lukeshu): In v3alpha2, change all of the `{foo}s`/`SecondDuration`
```

Comment:
Agree with the TODO, but do you care if this description is user-visible to anyone reading our CRDs?

Reply:
I'm not in love with them looking like that, but IMO it's still good for this info to be in the user-facing docs; it lets users know that something will be changing in an upcoming version.